sdks/v4-sdk/src/PositionManager.test.ts

```diff
@@ -8,6 +8,7 @@ import {
   SQRT_PRICE_1_1,
   TICK_SPACINGS,
   ZERO_LIQUIDITY,
+  PositionFunctions,
```

Comment:
why are the test files stored in `src`? is there a reason?

Reply:
yea not sure, this is the pattern from v3-sdk

Reply:
weirddd